wrapper.h

```diff
@@ -1,3 +1,10 @@
+
+#include <spdk/assert.h>
+
+#ifndef assert
```

Comment:
Could we include the "common" here rather than "harcode" again?

Reply:
Yes indeed, I will fix.